crates/bevy_asset/src/server/mod.rs

```diff
@@ -270,7 +270,26 @@ impl AssetServer {
     /// The asset load will fail and an error will be printed to the logs if the asset stored at `path` is not of type `A`.
     #[must_use = "not using the returned strong handle may result in the unexpected release of the asset"]
     pub fn load<'a, A: Asset>(&self, path: impl Into<AssetPath<'a>>) -> Handle<A> {
-        self.load_with_meta_transform(path, None)
+        self.load_with_meta_transform(path, None, ())
+    }
+
+    /// Begins loading an [`Asset`] of type `A` stored at `path` while holding a handle like a semaphore guard.
```

Comment:
"handle" is already very used as a word in Bevy assets, could you try to use another one?

Reply:
Rephrased to `guard item`.